### ISSUE TITLE
User feedback feature for missing items/wrong location/no inventory, etc. 

### ISSUE THREAD

**Issue by Ye-walmart** [Ye-walmart/Leapfrog_FMI#25]
https://github.com/Wiredcraft/walmart-find-my-item/issues/74

I see Karen has a mockup for this user feedback feature? Could we implement it before releasing to users (April 6)?

**Comment by fuzhuang** [comment]
@Ye-walmart Backend is not fully ready on this one, we also need to test if the data has been properly recorded. Suggest we push this item to the next release after MVP. cc @makara

**Comment by makara** [comment]
Backend is almost there, just some spec details I'll need to discuss with our frontend people. But I agree with @fuzhuang for pushing it to the next release, cause:

- We have another priority which is collecting user behavior events and generating reports. These all belong to marketing solutions and we prefer releasing things one after another.
- Things other than the search are not core features and we want to be careful, to not bringing noises or breaking UX.

but it's your call @Ye-walmart. What do you think?

**Comment by Ye-walmart** [comment]
Agree, collecting user behavior and generating reports are more important. Please focus on these as priority.
